connection.js

```diff
@@ -100,6 +100,7 @@ class Connection {
         this.transaction = null;
         this.tran_count = 0;
         this.capabilities = null;
+        this.ehlo_hello_message = config.get('ehlo_hello_message') || null;
```

Comment:
This is an improvement, but wouldn't this be better?

```js
this.ehlo_hello_message = config.get('ehlo_hello_message') || 'Haraka is at your service.'
```

Then the two ternaries below are simple variable references.